src/ecs/commands/runCommandInContainer.ts

```diff
@@ -0,0 +1,134 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import * as nls from 'vscode-nls'
+const localize = nls.loadMessageBundle()
+
+import * as vscode from 'vscode'
+import * as picker from '../../shared/ui/picker'
+import { Window } from '../../shared/vscode/window'
+import { getLogger } from '../../shared/logger'
+import { ChildProcess } from '../../shared/utilities/childProcess'
+import { EcsContainerNode } from '../explorer/ecsContainerNode'
+import { recordEcsRunExecuteCommand } from '../../shared/telemetry/telemetry.gen'
+import { DefaultSettingsConfiguration } from '../../shared/settingsConfiguration'
+import { extensionSettingsPrefix } from '../../shared/constants'
+
+export async function runCommandInContainer(node: EcsContainerNode, window = Window.vscode()): Promise<void> {
+    getLogger().debug('RunCommandInContainer called for: %O', node)
+
+    const verifySSMPluginResponse = await new ChildProcess(true, 'session-manager-plugin').run()
+    if (!verifySSMPluginResponse.stdout.startsWith('The Session Manager plugin was installed successfully')) {
```

Comment:
Dunno if we can rely on this output message. It should be enough to check that the exitCode === 0.

Reply:
done